x-pack/plugin/sql/jdbc/src/main/java/org/elasticsearch/xpack/sql/jdbc/jdbc/JdbcResultSet.java

```diff
@@ -344,7 +344,7 @@ public Object getObject(int columnIndex) throws SQLException {
             throw new SQLException("type is null");
         }
 
-        return getObject(columnIndex, type);
+        return convert(columnIndex, type);
```

Comment:
I think this includes the jdbc fix, right?

Reply:
Yes, that's the one (that triggers the StackOverflow).